test/Dialect/Seq/firreg.mlir

```diff
@@ -0,0 +1,654 @@
+// RUN: circt-opt %s -verify-diagnostics --lower-seq-to-sv -cse -canonicalize | FileCheck %s
```

Comment:
I think it would be best to avoid using CSE and Canonicalization in this test, so you can be sure you're testing the effects of lower-seq-to-sv, not the other passes.

Reply:
Done